### PR TITLE
CI : remove mention to wxWidgets in MAC build-dependencies

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Rationale behind Changes

remove mention to wxWidgets in MAC build-dependencies
### Suggested Testing Steps

watch ci

### CODE REVIEW

**Comment by Florin9doi** [comment]
also here: 
https://github.com/PCSX2/pcsx2/blob/227c2563c5bfac9eeaa3a442b9af8455ba6abff8/cmake/BuildParameters.cmake#L318

**Comment by Mrlinkwii** [comment]
> also here:
> 
> https://github.com/PCSX2/pcsx2/blob/227c2563c5bfac9eeaa3a442b9af8455ba6abff8/cmake/BuildParameters.cmake#L318

im just doing build-dependencies.sh file tellow etc can take care of that

**Comment by stenzek** [comment]
`QT_BUILD` is still relevant, one day I'll PR my nogui interface (which can be built alongside or instead of qt).

File: .github/workflows/scripts/macos/build-dependencies.sh [hunk 1]
```diff
@@ -2,11 +2,8 @@
 
 set -e
 
-if [ "$GUI" == "Qt" ]; then
 	export MACOSX_DEPLOYMENT_TARGET=10.14
```
Comment (TellowKrinkle) [inline .github/workflows/scripts/macos/build-dependencies.sh:5]:
```suggestion
export MACOSX_DEPLOYMENT_TARGET=10.14
```

Comment (Mrlinkwii) [reply .github/workflows/scripts/macos/build-dependencies.sh]: done

**Review by TellowKrinkle — APPROVED** [review]

**Pull request merged**